internal/cluster/validations/validations.go

```diff
@@ -501,7 +502,7 @@ func ValidateDiskEncryptionParams(diskEncryptionParams *models.DiskEncryption, D
 		if diskEncryptionParams.TangServers == "" {
 			return errors.New("Setting Tang mode but tang_servers isn't set")
 		}
-		tangServers, err := common.UnmarshalTangServers(diskEncryptionParams.TangServers)
```

Comment:
most unmarshal helpers are actually on common? any reason to move it?

Reply:
I moved it out of `/internal` because I needed this function for the agent code as well.
See [CheckTangConnectivity()](https://github.com/openshift/assisted-installer-agent/pull/341/files#diff-d1bab7abd87cd22a53866c28c672fb7386f432d9c44353543f3e3a4021afd987R74) in https://github.com/openshift/assisted-installer-agent/pull/341